src/installer/windows/index.ts

```diff
@@ -11,17 +11,18 @@ import {Installation} from './installation'
 
 export class WindowsToolchainInstaller extends VerifyingToolchainInstaller<WindowsToolchainSnapshot> {
   private get vsRequirement() {
-    const reccommended = '10.0.17763'
+    const recommended = '10.0.19041'
```

Comment:
The current logic uses the grater SDK version between current platform and `10.0.17763`. So for Windows 11 version `10.0.22621`, that version SDK will be chosen but if action is used on a Windows version lesser than `10.0.17763` then SDK of `10.0.17763` will be used.

This variable should be the lowest SDK version supported by Swift, not the latest one. Otherwise action will download the SDK unnecessarily, when it can reuse pre-installed SDK version.